py/moq-lite/tests/test_local.py

```
@@ -228,3 +228,259 @@ async def test_announced_broadcast():
+    group = track.append_group()
+    group.write_frame(b"")
+    group.finish()
+
+
+def test_raw_group_write_after_finish_fails():
+    broadcast = moq.BroadcastProducer()
+    track = broadcast.publish_raw("t")
+    group = track.append_group()
+    group.finish()
+
+    with pytest.raises(Exception):
```

Ruff (0.15.10)

[warning] Do not assert blind exception: `Exception`

(B017)